tldw_Server_API/app/api/v1/endpoints/admin/admin_identity_providers.py

```
@@ -0,0 +1,328 @@
+    )
+
+
+@router.get(
+    "/identity/providers",
+    response_model=IdentityProviderListResponse,
+    dependencies=[Depends(get_auth_principal), Depends(check_rate_limit)],
+)
+async def admin_list_identity_providers(
+    owner_scope_type: str | None = Query(None),
+    owner_scope_id: int | None = Query(None, ge=1),
+    enabled: bool | None = Query(None),
```

Ruff (0.15.6)

[warning] Boolean-typed positional argument in function definition

(FBT001)